license-details/README.md

```diff
@@ -0,0 +1,138 @@
+# Introduction
```

Comment:
I would not create a new folder for this, that seems like waste of space. Instead this should be part of docs.

Reply:
If we have any docs related to writing downstream backends, this might go near them, or at least be linked from them. I think putting it into `docs` makes sense.

Reply:
In commit 12 I moved all of the new files into the `docs` directory, with file names beginning with `licenses`